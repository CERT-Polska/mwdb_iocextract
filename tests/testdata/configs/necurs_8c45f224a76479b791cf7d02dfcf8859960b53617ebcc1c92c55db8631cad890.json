{"type": "necurs", "campaign_id": 5, "mutex": "NitrGB", "udp_dword": 78269159, "domains": [{"cnc": "178.32.31.41"}, {"cnc": "95.211.195.245"}, {"cnc": "94.231.81.244"}, {"cnc": "91.213.8.35"}, {"cnc": "151.236.6.6"}, {"cnc": "119.252.20.75"}, {"cnc": "198.100.146.51"}, {"cnc": "192.121.170.170"}, {"cnc": "78.47.34.12"}, {"cnc": "108.61.210.58"}, {"cnc": "109.69.8.34"}, {"cnc": "87.98.175.85"}, {"cnc": "106.186.17.181"}, {"cnc": "107.170.95.180"}, {"cnc": "npkxghmoru.biz"}], "udp_public_key": "-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApaozeDBHiT/mQkyGrdvp\nfx7nqAZe2RFM7xTTcPdteXLMMlZSoSN3yFjtjN8fU8N3zFypXg3DgCcxb+zHQe7f\ngLXjJfh09UGXFWSPpVK5V+OUKi6U58XaTT8L/GVnAsFbrUxx7y+Ziy6p41hJ1mnQ\nO3ya1CYaxlqT2mWPf/LQOHu0MN2CCfII2zSw+WiDeKfKAEQjOIBEh3qcLoYi2sid\nz2eqFongq3rzTVMwf2BaRH5Qz5GbCteWTBi7Bb65a+mNo+zGZVJL69HPLXaop9oJ\n5KR4zIXWaVwHJHT6ZkD5/TCpTzPrLve9woVb2CvFRBCObWJpasvK/1+NShdJBKpK\n5wIDAQAB\n-----END PUBLIC KEY-----", "url_path": "/forum/db.php", "c2_public_key": "-----BEGIN PUBLIC KEY-----\nMIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDHxpBfq7Iidiiv1uowBNWS/bD2\nzu1GjXOQQgA9Icxq+tSQmL8a1n012+5TfEoFsA85rblm853smCFdMVtThuVSomdJ\nP5frpNeomtKuR+74Ci0FmReh9sxiXW/eC665pemNrLnxao5gWTAgBR8WcCQB2A8N\niPrPL7XNh9i8NrtggwIDAQAB\n-----END PUBLIC KEY-----", "pub_dword": 918249603, "peers": {"in-blob": "2c7016dd5e9a5dbea706fe26cdc0071186a8d7003e34265630ffd3cb159ceb3a"}}