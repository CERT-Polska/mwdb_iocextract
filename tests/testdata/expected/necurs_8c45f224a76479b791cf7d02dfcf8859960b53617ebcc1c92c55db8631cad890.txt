RsaKey n=140287225539891372288370163333383608191668308305574916287484657625128871155095786217555437375383528205257458947995417410242029209737202821882501372407767882058722372147852820605589972732547507782844862241052606176197024551529336766328510975877123555576602312784368167182467079098159138841895773647984788201603 e=65537
NetLoc 178.32.31.41/
NetLoc 95.211.195.245/
NetLoc 94.231.81.244/
NetLoc 91.213.8.35/
NetLoc 151.236.6.6/
NetLoc 119.252.20.75/
NetLoc 198.100.146.51/
NetLoc 192.121.170.170/
NetLoc 78.47.34.12/
NetLoc 108.61.210.58/
NetLoc 109.69.8.34/
NetLoc 87.98.175.85/
NetLoc 106.186.17.181/
NetLoc 107.170.95.180/
NetLoc npkxghmoru.biz/
Mutex NitrGB
